### PR TITLE
update bcftools call wrapper

### PR DESCRIPTION
**Pull request by mbhall88**
See https://github.com/snakemake/snakemake-wrappers/pull/145#issue-455441639 for reasoning.  

Also, there was no pre-existing test for this wrapper.

### CODE REVIEW

File: bio/bcftools/call/wrapper.py [hunk 1]
```diff
@@ -7,8 +7,22 @@
...
+    pass
+
+
+valid_caller_opts = {"-c", "--consensus-caller", "-m", "--multiallelic-caller"}
+
+caller_opt = snakemake.params.get("caller", "")
+if caller_opt.strip() not in valid_caller_opts:
+    raise CallerOptionError(
+        "bcftools call expects either -m/--multiallelic-caller or "
+        "-c/--consensus-caller as caller options."
```
Comment (dlaehnemann) [inline bio/bcftools/call/wrapper.py:20]: Nit-picking here: `option.` without the `s`, I think.

**Comment by mbhall88** [comment]
@dlaehnemann change made

**Comment by dlaehnemann** [comment]
@mbhall88 Do you need this merged quickly? Otherwise I'd ask @johanneskoester to comment, first.

**Comment by mbhall88** [comment]
> @mbhall88 Do you need this merged quickly? Otherwise I'd ask @johanneskoester to comment, first.

No rush. Happy to get comments from Johannes.

**Pull request merged**